cherry_picker/cherry_picker.py

```diff
@@ -245,6 +245,9 @@ def continue_cherry_pick(self):
 
             self.cleanup_branch(cherry_pick_branch)
 
+            click.echo("\nBackport PR:\n")
+            click.echo(updated_commit_message)
```

Comment:
This prints out the title, prefixed with [X.Y], message body, and `cherry picked from .. ` message

Example output when backporting PR python/cpython#879:
```
[3.6] bpo-29931 fix __lt__ check in ipaddress.ip_interface for both v4 and v6. (GH-879)                     
                                                                                                            
the original logic was just comparing the network address                                                   
but this is wrong because if the network address is equal then                                              
we need to compare the ip address for breaking the tie                                                      
                                                                                                            
add more ip_interface comparison tests.                                                                     
(cherry picked from commit 7bd8d3e794782582a4ad1c9749424fff86802c3e)  
```

I recorded an asciicast: https://asciinema.org/a/123496